konfera/templates/konfera/registration_form.html

```diff
@@ -1,10 +1,14 @@
-{% extends 'konfera/base.html' %}{% load i18n %}
+{% extends 'konfera/base.html' %}
+{% load i18n %}
+{% load custom_filters %}
 
 {% block content_top %}
     <h2>{% blocktrans %} Registration of a {{ type }} {% endblocktrans %}</h2>
 {% endblock %}
 
 {% block content_details %}
+    <p class="list-group-item-text">{{ description }}</p>
```

Comment:
Coul you please do the style same as on `/{event}/ticket` page? e.g. there is also ticket name, and everything is grouped in div `list-group`.